### DISCUSSION TITLE
A strange problem since version 1.3.0.1000

### DISCUSSION THREAD

**Discussion by Digi2025** [dokan-dev/dokany#1237] [Q&A]
Hello
   There is a strange problem since version 1.3.0.1000 of dokan （before 1.2.2.1000 is OK ）. Once a local drive letter is mounted as network driver using mirror.exe , then copying, pasting or deleting files from the network driver, the system copy UI progress bar will be stuck in the last 99% for a few seconds, and this problem will only appear after the system restarts,mounted in first time, the first file operated will be like this, and the rest files operation will be normal.
  By the way, this problem will only appear when mounting it as a network disk. Is there any ideas to fix it ?

Thanks !

**Comment by Liryna** [comment]
Hi @Digi2025 ,

I have never seen the behavior but it could happen if antivirus or else are hanging the copy for analysis purpose. Have you tried to use a later version of Dokan v1 or even v2 ? 

You can list the possible filter drivers (antivirus etc) that might be interacting here with this command line: `fltmc filters`

**Reply by Digi2025** [reply]
We did the test on v2 verison,but also the same problem .
